src/main/java/com/cag/cagbackendapi/daos/impl/UserDao.kt

```diff
@@ -46,9 +46,9 @@ class UserDao : UserDaoI {
 
         val userEntity = userRepository.getByUserId(userId) ?: return null
 
-        userEntity.setFirstName(userUpdateDto.first_name)
-        userEntity.setLastName(userUpdateDto.last_name)
-        userEntity.setEmailJava(userUpdateDto.email)
+        userEntity.first_name = userUpdateDto.first_name
+        userEntity.last_name = userUpdateDto.last_name
+        userEntity.email = userUpdateDto.email
```

Comment:
should we not have these variables be private and only use getters/setters? Or is this a Kotlin thing that's valid

Reply:
This is a kotlin thing, the dot notation behind the scenes is doing setters and getters. Your head is in the right place!